docs/rules/jsx-no-target-blank.md

```diff
@@ -6,8 +6,8 @@
 
 <!-- end auto-generated rule header -->
 
-When creating a JSX element that has an `a` tag, it is often desired to have the link open in a new tab using the `target='_blank'` attribute. Using this attribute unaccompanied by `rel='noreferrer'`, however, is a severe security vulnerability (see [noreferrer docs](https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer) and [noopener docs](https://html.spec.whatwg.org/multipage/links.html#link-type-noopener) for more details)
-This rules requires that you accompany `target='_blank'` attributes with `rel='noreferrer'`.
+When creating a JSX element with an `a` tag, the link is often desired to open in a new tab using the `target='_blank'` attribute. Using this attribute unaccompanied by `rel='noreferrer'`, however, is a severe security vulnerability (see [noreferrer docs](https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer) and [noopener docs](https://html.spec.whatwg.org/multipage/links.html#link-type-noopener) for more details).
```

Comment:
"the link is often desired to open" reads wrong to me; the "it" isn't referring to "the link" but to "a generic person"